src/Integration.Vsix.UnitTests/SonarLintTagger/CancellableJobRunnerTests.cs

```diff
@@ -117,32 +115,32 @@ public void AllOperationsExecuted()
         public void CancelAfterFirstOperation()
         {
             // Arrange
-            var testLogger = new TestLogger(logToConsole: true);
-            TestContext.WriteLine($"Test executing on thread {Thread.CurrentThread.ManagedThreadId}");
-            var progressRecorder = new ProgressNotificationRecorder();
+            var testLogger = new TestLogger(logToConsole: true, logThreadId: true);
+            testLogger.WriteLine("[Test] Executing test");
+            var progressRecorder = new ProgressNotificationRecorder(testLogger);
 
             bool op1Executed = false, op2Executed = false;
             CancellableJobRunner testSubject = null;
 
             Action op1 = () =>
             {
-                TestContext.WriteLine($"Executing op1 on thread {Thread.CurrentThread.ManagedThreadId}");
+                testLogger.WriteLine("[Test] Executing op1");
                 op1Executed = true;
 
                 testSubject.Cancel();
             };
 
             Action op2 = () =>
             {
-                TestContext.WriteLine($"Executing op2 on thread {Thread.CurrentThread.ManagedThreadId}");
+                testLogger.WriteLine("[Test] Executing op2");
```

Comment:
Maybe add the [Test] prefix as part of the testLogger class?

Reply:
I wanted to differentiate between messages that are logged by the test code and messages that are logged by the product code.